apps/table.html

```diff
@@ -36,11 +37,18 @@
 				}
 
 				//Checking if silde with given name already exists
-				if(existing_slide_names.includes(slide.value)) {
+				if(existingSlideNames.includes(slide.value)) {
 					changeStatus("UPLOAD", "Slide with given name already exists");
 					return false;
 				}
 			}
+			//Checking for extinsion
+			let filename = document.getElementById("filename0").value
+			var fileExtension = filename.toLowerCase().split('.').reverse()[0];
+			if(!allowedExtensions.includes(fileExtension)) {
+				changeStatus("UPLOAD", fileExtension + " files are not allowed");
```

Comment:
```suggestion
				changeStatus("UPLOAD", fileExtension + " files are not compatible");
```
A bit more descriptive?

Reply:
Yes. Sure!